tests/cheetah/remove_call_data/test_can_remove_calls/multiline.tmpl

```diff
@@ -1,7 +1,5 @@
         foo  bar
-        foo $dotted.name bar
+        foo ${dotted.name} bar
         foo $cheetahvar bar
         foo $cheetah.var bar
-        foo $(
-            x.upper() for x in mylist
-        ) bar
```

Comment:
I believe the point of this testcase was the multiline expression.
Isn't that valid anymore?